.env

```diff
@@ -0,0 +1,2 @@
+MERRITT_USER=tjohnson
+MERRITT_PASSWORD=fakepwd
```

Comment:
should this file be `.env.development`?

Reply:
No this is for the `test` environment.`dotenv` gets loaded in [`spec_helper.rb`](https://github.com/ucsblibrary/alexandria/pull/183/files#diff-93830fa29d616f7c87903d08b5b1b29aR23)

Reply:
Gotcha.  For Starlight we use `.env.test` and `.env.development` so it would be nice to mirror that: https://gitlab.com/surfliner/surfliner/tree/master/starlight

Reply:
I will let you refactor this when you work on ADRL next. I am running against a tight deadline and can't afford the deviation from the task at hand.